scripts/test/ProcessManagement/TransitionAction/AppointmentRemove.t

```diff
@@ -0,0 +1,135 @@
+# --
+# Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# --
+# This software comes with ABSOLUTELY NO WARRANTY. For details, see
+# the enclosed file COPYING for license information (AGPL). If you
+# did not receive this file, see http://www.gnu.org/licenses/agpl.txt.
+# --
+
+use strict;
+use warnings;
+use utf8;
+
+use vars (qw($Self));
+
+use Kernel::System::VariableCheck qw(:all);
+
+$Kernel::OM->ObjectParamAdd(
+    'Kernel::System::UnitTest::Helper' => {
+        RestoreDatabase => 1,
+    },
+);
+
+my $AppointmentObject  = $Kernel::OM->Get('Kernel::System::Calendar::Appointment');
+my $CalendarObject     = $Kernel::OM->Get('Kernel::System::Calendar');
+my $ConfigObject       = $Kernel::OM->Get('Kernel::Config');
+my $GroupObject        = $Kernel::OM->Get('Kernel::System::Group');
+my $HelperObject       = $Kernel::OM->Get('Kernel::System::UnitTest::Helper');
+my $TicketObject       = $Kernel::OM->Get('Kernel::System::Ticket');
+my $ZnunyHelperObject  = $Kernel::OM->Get('Kernel::System::ZnunyHelper');
+my $DynamicFieldObject = $Kernel::OM->Get('Kernel::System::DynamicField');
+my $BackendObject      = $Kernel::OM->Get('Kernel::System::DynamicField::Backend');
+
+my $TransitionActionObject = $Kernel::OM->Get('Kernel::System::ProcessManagement::TransitionAction::AppointmentRemove');
+
+my $GroupID = $GroupObject->GroupLookup(
+    Group => 'users',
+);
+
+my %Calendar = $CalendarObject->CalendarCreate(
+    CalendarName => 'Calendar 1',
```

Comment:
If the calendar already exists, you will have a problem. 
Either you update the calendar or you use RandomID in Name etc.